pkg/addons/applier.go

```diff
@@ -101,6 +102,19 @@ func (a *Applier) GetAirgapCharts() ([]v1beta1.Chart, []v1beta1.Repository, erro
 	return regChart, regRepo, nil
 }
 
+func (a *Applier) GetAirgapImages() ([]string, error) {
```

Comment:
how about naming it the same here as well?
```suggestion
func (a *Applier) GetAdditionalImages() ([]string, error) {
```